terraform/paas/route.tf

```diff
@@ -15,3 +21,4 @@ locals {
   app_endpoint = "${cloudfoundry_route.route_cloud.hostname}.${data.cloudfoundry_domain.cloudapps.name}"
 }
 
+
```

Comment:
Extra blank line?

Reply:
Yes they were put in to force the files I had checked in earlier to look like they have changed, so it makes the review easier